rotkehlchen/tests/unit/accounting/evm/test_aave.py

```diff
@@ -0,0 +1,296 @@
+from typing import TYPE_CHECKING
+
+import pytest
+
+from rotkehlchen.accounting.cost_basis.base import (
+    AssetAcquisitionEvent,
+    CostBasisInfo,
+    MatchedAcquisition,
+)
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.evm_event import EvmEvent
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.ethereum.modules.aave.constants import CPT_AAVE_V2
+from rotkehlchen.chain.ethereum.modules.aave.v2.accountant import Aavev2Accountant
+from rotkehlchen.constants.assets import A_DAI, A_REN
+from rotkehlchen.constants.misc import ONE, ZERO
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.factories import make_evm_address, make_evm_tx_hash
+from rotkehlchen.types import Location, Price, Timestamp
+from rotkehlchen.utils.misc import ts_sec_to_ms
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.accountant import Accountant
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+
+TS1, TS2 = Timestamp(1), Timestamp(2)
+TSMS1, TSMS2 = ts_sec_to_ms(TS1), ts_sec_to_ms(TS2)
+USER_ADDRESS = make_evm_address()
+HASH1, HASH2 = make_evm_tx_hash(), make_evm_tx_hash()
+MOCKED_PRICES = {
+    A_DAI.identifier: {
+        'EUR': {
+            TS1: Price(ONE),
+            TS2: Price(ONE),
+        },
+    }, Asset('eip155:1/erc20:0x028171bCA77440897B824Ca71D1c56caC55b68A3'): {  # ADAI
+        'EUR': {
+            TS1: Price(ONE),
+            TS2: Price(ONE),
+        },
+    }, Asset('eip155:1/erc20:0xcd9D82d33bd737De215cDac57FE2F7f04DF77FE0'): {  # variableDebtREN
+        'EUR': {
+            TS1: Price(ONE),
+            TS2: Price(ONE),
+        },
+    }, A_REN.identifier: {
+        'EUR': {
+            TS1: Price(ONE),
+            TS2: Price(ONE),
+        },
+    },
+}
```

Comment:
if the price is always one why not use 

```
@pytest.mark.parametrize('default_mock_price_value', [ONE])
```

Reply:
Hmm ehm ... I started trying to give out a bit more complicated prices but ended up keeping it simple. Your suggestion is probably simpler

Reply:
done